GenderPayGap.WebUI/Views/AccountCreation/AlreadyCreatedAnAccountQuestion.cshtml

```diff
@@ -0,0 +1,99 @@
+﻿@using GenderPayGap.WebUI.Models.AccountCreation
+@using GovUkDesignSystem
+@using GovUkDesignSystem.Attributes
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.AccountCreation.AlreadyCreatedAnAccountViewModel
+@{
+    ViewBag.Title = "Have you already created your user account - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{ await Html.RenderPartialAsync("/Views/ReportingStepByStep/ReportingPageHeader.cshtml"); }
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">Have you already created your user account?</h1>
+    </div>
+</div>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <form method="GET" action="@Url.Action("AlreadyCreatedAnAccountQuestionGet", "AccountCreation")">
+
+            <div class="govuk-form-group">
+
+                @(Html.GovUkErrorSummary(new[]
+                {
+                    nameof(AlreadyCreatedAnAccountViewModel.HaveYouAlreadyCreatedYourUserAccount)
+                }))
```

Comment:
(unimportant comment) The property names (like 29) is only there so the errors appear in the right order.
If there is only one property, you can omit this list, because it won't make any difference.

Reply:
Done